src/main/java/com/dumbdogdiner/stickyapi/common/nbt/NbtBooleanTag.java

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright (c) 2020-2021 DumbDogDiner <dumbdogdiner.com>. All rights reserved.
+ * Licensed under the MIT license, see LICENSE for more information...
+ */
+package com.dumbdogdiner.stickyapi.common.nbt;
+
+import com.google.common.base.Preconditions;
+import com.google.gson.JsonElement;
+import com.google.gson.JsonPrimitive;
+import org.jetbrains.annotations.NotNull;
+
+
+/**
+ * A convenience class for converting booleans from {@link JsonPrimitive}s, and for easily creating
+ */
+public class NbtBooleanTag extends NbtPrimitiveTag<Boolean> {
```

Comment:
```suggestion
public class NBTBooleanTag extends NBTPrimitiveTag<Boolean> {
```
This is personal preference I guess, but having abbreviations in class names be uppercase appears cleaner imo